assets/helpers/user/user.js

```diff
@@ -66,8 +68,12 @@ const init = (dispatch: Function) => {
             dispatch(setFullName(data.name));
           }
           if (data.email) {
+            dispatch(setIsSignedIn(true));
```

Comment:
I've put this here for those who appear to be signed on...setting them to be signed in based off whether the response has an email seemed like the most sensible option, as if the response from identity has an email field then they must be signed in. You could say the same about the other fields too though, but it has to go somewhere.